api/preprints/serializers.py

```diff
@@ -177,6 +184,10 @@ class PreprintSerializer(TaxonomizableSerializerMixin, MetricsSerializerMixin, J
         },
     )
 
+    has_data_links = ser.NullBooleanField(required=False)
+    why_no_data = ser.CharField(required=False, allow_blank=True, allow_null=True)
+    data_links = ser.ListField(required=False)
```

Comment:
Should this also have `child=ser.URLField` to keep it consistent with the model?